src/web/releases.rs

```diff
@@ -605,26 +608,30 @@ pub fn activity_handler(req: &mut Request) -> IronResult<Response> {
 
 pub fn build_queue_handler(req: &mut Request) -> IronResult<Response> {
     let conn = extension!(req, Pool).get();
-    let mut crates: Vec<(String, String, i32)> = Vec::new();
-    for krate in &conn
+    let query = conn
         .query(
             "SELECT name, version, priority
-                              FROM queue
-                              WHERE attempt < 5
-                              ORDER BY priority ASC, attempt ASC, id ASC",
+                          FROM queue
+                          WHERE attempt < 5
+                          ORDER BY priority ASC, attempt ASC, id ASC",
```

Comment:
If we're going to fix the formatting, let's fix it properly :laughing: 
```suggestion
             "SELECT name, version, priority 
              FROM queue
              WHERE attempt < 5
              ORDER BY priority ASC, attempt ASC, id ASC",
```